depth_clustering:
  centerz_threshold: 2.0
  dis2d_threshold: 3.0
  min_cluster_size: 50
  max_cluster_size: 1000
  smooth_window_size: 7
  ground_remove_angle_d: 10

lidar:
  is_mid360_custom_msg: true

odometry:
  is_use_odometry: true
  is_mid360_tf: true
  space_width: 5
  space_length: 18
  lidar2robot_x: 0.0
  lidar2robot_y: 0.24215

topic: 
  odom_topic: "/aft_mapped_to_init"
  lidar_topic: "/livox/lidar"


